### ISSUE TITLE
Bug with checkm profile -- and possible fix

### ISSUE THREAD

**Issue by tcb72** [Ecogenomics/CheckM#273]
Hello,

Running checkm v. 1.1.2, installed via Conda. 

Doing some relative abundance calculations using a combination of `checkm coverage` + `checkm profile`. However, when running `checkm profile`, I get this error:

```
[2020-08-10 10:27:23] INFO: CheckM v1.1.2
[2020-08-10 10:27:23] INFO: checkm profile -f profile.txt checkm_coverage_output.tsv
[2020-08-10 10:27:23] INFO: [CheckM - profile] Calculating percentage of reads mapped to each bin.
[2020-08-10 10:27:23] INFO: Determining number of reads mapped to each bin.
Traceback (most recent call last):
  File "/home/tbiondi/.conda/envs/checkm/bin/checkm", line 611, in <module>
    checkmParser.parseOptions(args)
  File "/home/tbiondi/.conda/envs/checkm/lib/python3.7/site-packages/checkm/main.py", line 895, in parseOptions
    self.profile(options)
  File "/home/tbiondi/.conda/envs/checkm/lib/python3.7/site-packages/checkm/main.py", line 674, in profile
    profile.run(options.coverage_file, options.file, options.bTabTable)
  File "/home/tbiondi/.conda/envs/checkm/lib/python3.7/site-packages/checkm/profile.py", line 109, in run
    pTable = prettytable.PrettyTable(header)
NameError: name 'prettytable' is not defined
```

THE FIX: 

The issue points towards line 109 in profile.py (as seen above.) The fix is as follows: In profile.py, change

```
import checkm.prettytable
```

to 

```
import checkm.prettytable as prettytable
```

Then it works fine.

This is the same problem mentioned in #258, which was closed.

**Comment by jjacobson95** [comment]
Still not fixed. Created a pull request.

**Comment by dparks1134** [comment]
Sorry for the slow fix.  This issue has been resolved in CheckM v1.1.7 which will be released shortly.
